Remotion/Web/Development.WebTesting.IntegrationTests/TreeViewControlObjectTest.cs

```diff
@@ -94,7 +94,7 @@ public void TestNodeIsChecked ()
       var backupTimeout = rootNode.Scope.ElementFinder.Options.Timeout;
       rootNode.Scope.ElementFinder.Options.Timeout = TimeSpan.Zero;
 
-      Assert.That (() => rootNode.IsChecked(), Throws.InstanceOf<MissingHtmlException>());
+      Assert.That (() => rootNode.IsChecked(), Throws.InstanceOf<WebTestException>());

```

Comment:
Having a tree node but not being able to see whether it's checked or not may be a `MissingHtmlException`.

Reply:
IF i understand the test correctly, this just means that there is no checkbox on the rootNode, Why the class design allows this to happen is another discussion. But im not 100% sure, maybe before completing the task lets look over the test and implementation again

Reply:
This should be a `WebTestException`.